crates/matrix-sdk-crypto/src/verification/machine.rs

```diff
@@ -318,10 +318,6 @@ impl VerificationMachine {
     ) -> Result<(), CryptoStoreError> {
         let event = event.into();
 
-        let span = tracing::Span::current();
-        span.record("event_type", display(event.event_type()));
-        span.record("sender", display(event.sender()));
```

Comment:
If we no longer need these, they should also not be mentioned in the `instrument` arguments.

Reply:
Do we want to keep the span, i.e. the `#[instrument]` attribute?

Reply:
🤷🏼

Reply:
Left it in for now. If we decide that it's useless we can remove it later.